core/interop/pinvoke/marshalling/src/MarshallingSampleNative/common.h

```diff
@@ -18,5 +18,11 @@
 #endif
 
 typedef int BOOL;
+#define TRUE 1
+#define FALSE 0
+
+typedef int VARIANT_BOOL;
```

Comment:
This should be `short` instead of `int`.

Reply:
shoot. Good call. Completely forgot that was the underlying type.